jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcExecutionRepository.java

```diff
@@ -98,14 +98,18 @@ protected Condition statesFilter(List<State.Type> state) {
             .in(state.stream().map(Enum::name).collect(Collectors.toList()));
     }
 
+    abstract protected Condition labelsFilter(Map<String, String> labels);
```

Comment:
You should hook into the `findCondition` method and adds label filter here as was done for the Flow.
You can check the `PostgresFlowRepository` for example.

Reply:
It got switched to `findCondition`